### ISSUE TITLE
The "Issue Opened Triage" GitHub action workflow is broken

### ISSUE THREAD

**Issue by primeos-work** [hashicorp/terraform-provider-kubernetes#1523]
After opening #1520 I got an email that the workflow run failed:
```
[hashicorp/terraform-provider-kubernetes] Run failed: Issue Opened Triage - main (269cba3)
```

Looking at https://github.com/hashicorp/terraform-provider-kubernetes/actions/workflows/issue-opened.yml we can see that it always fails. The issue is the following (there are multiple required inputs):
```
Error: Input required and not supplied: enable-versioned-regex
```

This required input is not required, see:
- https://github.com/github/issue-labeler/blob/0207bb58c165f7229d560291b28d8de891bc46d1/README.md
- https://github.com/github/issue-labeler/blob/f40302de168dcc01a3689fee53e89ad74ed0a9dd/action.yml

Before the last update (#1480) it failed like this btw:
```
Unable to resolve action `github/issue-labeler@v2`, unable to find version `v2`
```

I propose fixing or disabling that workflow as those emails can be confusing.

![image](https://user-images.githubusercontent.com/95224042/144267637-f4dc559f-8137-4294-bda4-b370a243a973.png)

**Comment by iBrandyJackson** [comment]
https://github.com/hashicorp/terraform-provider-kubernetes/pull/1517

**Comment by iBrandyJackson** [comment]
Closing issue, as this was fixed with #1517 . Thanks for bring this to our attention.
